src/tests/ftest/dfuse/mu_perms.py

```diff
@@ -303,11 +301,11 @@ def test_dfuse_mu_perms_il(self):
         env_without_il = verify_perms_cmd.env.copy()
         env_with_il = env_without_il.copy()
         env_with_il.update({
-            'LD_PRELOAD': os.path.join(self.prefix, 'lib64', 'libioil.so'),
+            'LD_PRELOAD': os.path.join(self.prefix, 'lib64', il_lib),
             'D_IL_REPORT': -1  # Log all intercepted calls
         })
 
-        def _verify(use_il, expected_il_messages, expect_der_no_perm):
+        def _verify(use_il, expected_il_messages, expect_der_no_perm, il_lib=None):
```

Comment:
This `_verify` function is defined in the same scope where `il_lib` exists, so there is no need to pass `il_lib` to this function. You can just use it
```suggestion
        def _verify(use_il, expected_il_messages, expect_der_no_perm):
```

Reply:
Thank you! Fixed as you suggested.